src/parser.c

```diff
@@ -170,13 +171,18 @@ const char* parse_annotation(
 
     return ptr;
 }
+/** Build error string*/
+void build_error_string(char *error, const char* signature, long position, const char *system_id){
+    sprintf(error, "\"%s\" at position %ld. System \"%s\"", signature, position, system_id);
```

Comment:
This looks good, already much more informative! What I'd suggest is to add a descriptive string and format it so that it looks a bit more parser-ish, something like this:

```
Move: 1: no such component "Position"
    Position, Velocity
    ^
```

Reply:
Sure. I need to think how to format.

Reply:
Done.